packages/EasyCore/src/Bridge/Symfony/ApiPlatform/OpenApi/Processor/UpdateDocsComponentsSchemasProcessor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\ApiPlatform\OpenApi\Processor;
+
+use Carbon\CarbonImmutable;
+use EonX\EasyCore\Bridge\Symfony\ApiPlatform\OpenApi\Provider\OpenApiContextProvider;
+
+final class UpdateDocsComponentsSchemasProcessor implements
+    DecorationProcessorInterface,
+    DefaultDecorationProcessorInterface
+{
+    public function __construct(private OpenApiContextProvider $openApiContextProvider)
+    {
+    }
+
+    public function process(array $documentation): array
+    {
+        $schema = [];
+
+        foreach ($documentation['components']['schemas'] as $schemaName => &$schema) {
+            if (isset($schema['properties']) === false) {
+                continue;
+            }
+
+            $this->updateDocsPropertyDescriptions($schema['properties']);
+
+            $this->updateDocsPropertyLinks($schema['properties']);
+
+            $schema = $this->openApiContextProvider->getCustomSchemaProperties($schemaName, $schema);
+        }
+        unset($schema);
+
+        return $documentation;
+    }
+
+    /**
+     * @param mixed[] $properties
+     */
+    private function updateDocsPropertyDescriptions(array &$properties): void
+    {
+        $property = [];
+
+        foreach ($properties as $propertyName => &$property) {
+            // API Platform doesn't use the embedded descriptions for createdAt/updatedAt
+            if ($propertyName === 'createdAt') {
+                $property['description'] = 'Timestamp of resource creation.';
+            }
+            if ($propertyName === 'updatedAt') {
+                $property['description'] = 'Timestamp of the most recent resource update.';
+            }
+
+            // Remove newlines in property descriptions (except for before list items)
+            if (isset($property['description'])) {
+                $property['description'] = \preg_replace('/\\n([^-])/', ' $1', $property['description']);
+            }
+        }
+
+        unset($property);
+    }
+
+    /**
+     * @param mixed[] $properties
+     */
+    private function updateDocsPropertyLinks(array &$properties): void
+    {
+        $property = [];
+        $dateTimeExample = CarbonImmutable::now()->toIso8601String();
```

Comment:
Let's use here some exact date to make generated docs the same.

And let's add tests:
- two API resources, with all cases that we transform in our transformers.
- assert that generated JSON is the same.

Maybe we should think about creating the new package `easy-api-platform`, and move all related stuff there.